src/nplinker/metabolomics/ISpectrumLoader.py

```diff
@@ -0,0 +1,11 @@
+from abc import ABC, abstractmethod
+from typing import Iterable
+
+from nplinker.metabolomics.spectrum import Spectrum
+
+
+class ISpectrumLoader(ABC):
+    
+    @abstractmethod
+    def spectra(self) -> Iterable[Spectrum]:
```

Comment:
to narrow the type `Iterable` to `collections.abc.Sequence`?

Reply:
Done in 08d9d1b7de0a084a659868bb44df9a7863a7f145